### PR TITLE
Skip Python 3.5

### PR DESCRIPTION
**Pull request by ocefpaf**
Maybe we should also remove `.travis.yml`, `circle.yml` (and `ci_support/run_docker_build.sh`) like we did for `appveyor.yml` when there are no OS X and Linux builds.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,22 +13,25 @@ source:  # [win]
     fn: mingwpy-0.1.0b3-cp27-none-win_amd64.whl                                                       # [py27 and win64]
 
 build:
-    skip: True # [not win]
     number: 0
```
Comment (pelson) [inline recipe/meta.yaml]: You should probably bump the build number for a change of this size. Particularly as the runtime dependencies have changes (this is a functional change).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -13,22 +13,25 @@ source:  # [win]
...
 
 about:
     home: https://github.com/mingwpy/mingw-builds
-    license: 3-clause BSD
-    summary: 'GCC for Windows 64 & 32 bits'
+    license: BSD 3-clause
+    summary: GCC for Windows 64 & 32 bits
 
 extra:
     recipe-maintainers:
```
Comment (pelson) [inline recipe/meta.yaml:37]: Is there anybody else who we should be contacting about this recipe?

Comment (ocefpaf) [inline recipe/meta.yaml:37]: @carlkl I'd love to bring you to conda-forge!

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: Done.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,22 +13,25 @@ source:  # [win]
     fn: mingwpy-0.1.0b3-cp27-none-win_amd64.whl                                                       # [py27 and win64]
 
 build:
-    skip: True # [not win]
-    number: 0
+    number: 1
+    skip: True # [unix]
```
Comment (pelson) [inline recipe/meta.yaml:17]: I definitely prefer the `not win` - far more specific in this case.

Comment (ocefpaf) [inline recipe/meta.yaml:17]: I didn't change it for any special reason. I can revert it later.

Comment (pelson) [inline recipe/meta.yaml:17]: No rush (or even requirement to do it). Just wanted to share knowledge :smile:

**Pull request merged**